.github/workflows/build_and_test.yml

```
@@ -10,3 +10,18 @@ jobs:
       - run: npm install
       - run: npm test
       - run: npm run build
+  deploy-to-github:
+    runs-on: ubuntu-latest
+    steps:
```

need to limit to just `develop`

```suggestion
    if: ${{ github.ref == 'refs/heads/develop' }}
    steps:
```